src/pages/settings/Profile/PronounsPage.js

```diff
@@ -23,31 +23,21 @@ const defaultProps = {
     ...withCurrentUserPersonalDetailsDefaultProps,
 };
 
-function PronounsPage(props) {
-    const [initiallyFocusedOption, setInitiallyFocusedOption] = useState({});
-    const [searchValue, setSearchValue] = useState('');
-    const [pronounsList, setPronounsList] = useState([]);
+function PronounsPage({currentUserPersonalDetails, translate}) {
+    const currentPronouns = lodashGet(currentUserPersonalDetails, 'pronouns', '');
+    const currentPronounsKey = currentPronouns.substring(CONST.PRONOUNS.PREFIX.length);
+    const currentPronounsText = _.chain(translate('pronouns'))
```

Comment:
I think this will be calculated on every rerender, and it is being used just to init the `useState` worth moving into the `useState`?

Reply:
Great catch! fixed